### PR TITLE
OPENEUROPA-2145: Add spacing between paragraphs.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

Add spacing between the paragraphs.

Check ECL for an indication of spacing value that we can use and use that. Otherwise a default of 20px would be good.

This needs to go into oe_theme.

### CODE REVIEW

File: oe_theme.theme [hunk 1]
```diff
@@ -245,6 +245,18 @@ function oe_theme_preprocess_form_element(array &$variables): void {
...
+      $variables['field_name'] === 'paragraphs' &&
+      $variables['element']['#bundle'] === 'landing_page') ||
+    ($variables['entity_type'] === 'block_content' &&
+      $variables['field_name'] === 'field_content' &&
+      $variables['element']['#bundle'] === 'content_block')) {
+    foreach ($variables['items'] as &$item) {
+      $item['attributes']->addClass('ecl-u-mv-s');
+    }
+  }
+
```
Comment (ademarco) [inline oe_theme.theme]: We can't simply hardcode bundle names in the theme, this is a generic component.

File: oe_theme.theme [hunk 2]
```diff
@@ -245,6 +245,18 @@ function oe_theme_preprocess_form_element(array &$variables): void {
  * Add paragraph formatting to body.
  */
 function oe_theme_preprocess_field(&$variables, $hook): void {
+  // Add space between paragraphs.
+  if (($variables['entity_type'] === 'node' &&
+      $variables['field_name'] === 'paragraphs' &&
+      $variables['element']['#bundle'] === 'landing_page') ||
```
Comment (ademarco) [inline oe_theme.theme]: We can't simply hardcode bundle names in the theme, this is a generic component.

**Review by ademarco — DISMISSED** [review]

File: oe_theme.theme (hunk 2 above)
Comment (nagyad) [reply oe_theme.theme]: I think this should not be tied to any bundle, spacing has to be correct on any bundle that has paragraph field.

File: oe_theme.theme [hunk 3]
```diff
@@ -245,6 +245,18 @@ function oe_theme_preprocess_form_element(array &$variables): void {
  * Add paragraph formatting to body.
  */
 function oe_theme_preprocess_field(&$variables, $hook): void {
+  // Add space between paragraphs.
```
Comment (voidtek) [inline oe_theme.theme]: Could have more information concerning the class chose?
for example the link of the ECL2 marge documentation.

File: oe_theme.theme [hunk 4]
```diff
@@ -245,6 +245,19 @@ function oe_theme_preprocess_form_element(array &$variables): void {
  * Add paragraph formatting to body.
  */
 function oe_theme_preprocess_field(&$variables, $hook): void {
+  // Add space between field items of
+  // 'Entity reference revisions' field type (paragraphs for example).
+  // We have to avoid adding of margins for a nested paragraphs.
+  // For more info @see https://v1--europa-component-library.netlify.com/ec/components/detail/ec-utility-margin
```
Comment (nagyad) [inline oe_theme.theme]: Add margin between paragraph items.

**Review by nagyad — CHANGES_REQUESTED** [review]
image
As you can see on the image the content-row paragraph items are not containing the correct spacing.

**Review by nagyad — APPROVED** [review]

**Review by imanoleguskiza — APPROVED** [review]

**Pull request merged**